### DISCUSSION TITLE
Unable to read the CLA without connecting Github to Zed

### DISCUSSION THREAD

**Discussion by Erk-** [zed-industries/zed#4442] [Feature Requests]
It is currently not possible to read the Zed cla at https://zed.dev/cla without connecting with GitHub to Zed, and in that process handing over personal information such as Email without information about how it will be used.

**Comment by ZeroCool2u** [comment]
This is the content of the CLA as of now:


Zed Contributor License and Feedback Agreement
Welcome to our Contributor License and Feedback Agreement! Here's a quick breakdown of what's inside:

You (the contributor) are entering into an agreement with Zed Industries, Inc.
You're giving Zed permission to use and share your contributions (like original works or modifications).
You assure us that the contributions are truly your own and you have the legal right to share them.
You're not required to support your contributions but you're welcome to if you wish.
If you ever notice an error or change in the details you've given us, you agree to let us know.
In short, this agreement covers the terms for your valuable contributions to Zed's projects.

Legally binding section follows:
By submitting a Contribution, use of the Solution or any components related thereto (as such is further defined in Zed’s End User License Agreement located here: https://zed.dev/eula) or as enabled pursuant to your use of Zed’s collaboration tools offered therein, you hereby accept and agree to the terms and conditions set forth in this Contributor License and Feedback Agreement (the “Agreement”) for Your present and future Contributions submitted to Zed Industries, Inc. (“Company”). Except for the license granted herein to Company and recipients of software distributed or made available by Company, You reserve all right, title, and interest in and to Your Contributions.

1. Definitions.

“Contributor”, "You", and "Your" shall mean the copyright owner or legal entity authorized by the copyright owner that is making this Agreement with Company. For legal entities, the entity making a Contribution and all other entities that control, are controlled by, or are under common control with that entity are considered to be a single Contributor. For the purposes of this definition, "control" means (i) the power, direct or indirect, to cause the direction or management of such entity, whether by contract or otherwise, or (ii) ownership of fifty percent (50%) or more of the outstanding shares, or (iii) beneficial ownership of such entity.

"Contribution" shall mean Feedback (as defined below), any original work of authorship, and any modifications or additions to an existing work, that is intentionally submitted by You to Company for inclusion in, or documentation of, any of the products owned or managed by Company (the "Work"). For the purposes of this definition, "submitted" means any form of electronic, verbal, or written communication sent to Company or its representatives, including but not limited to communication on electronic mailing lists, source code control systems, and issue tracking or collaboration systems that are managed by, or on behalf of, Company for the purpose of discussing and improving the Work, but excluding communication that is conspicuously marked or otherwise designated in writing by You as "Not a Contribution."

“Feedback” means suggestions, comments, improvements, software code / snippets or other information submitted to, shared with or otherwise made available to Zed or its contributors with respect to, or in connection with the use or interaction with, the Work, Zed Network Based Service or Editor technology (as defined by Zed and as further described within Zed’s End User License Agreement located at https://zed.dev/eula).

2. Grant of Copyright License. Subject to the terms and conditions of this Agreement, You hereby grant to Company, and to recipients of software distributed by Company related hereto, a perpetual, worldwide, non-exclusive, no-charge, royalty-free, irrevocable copyright license to reproduce, prepare derivative works of, publicly display, publicly perform, sublicense, and distribute, Your Contributions and such derivative works (the “Contributor License Grant”). Further, to the extent that You participate in any livestream or other collaborative feedback generating session offered by Company, you hereby consent to use of any content shared by you in connection therewith in accordance with the foregoing Contributor License Grant.

3. Grant of Patent License. Subject to the terms and conditions of this Agreement, You hereby grant to Company and to recipients of software distributed by Company a perpetual, worldwide, non-exclusive, no-charge, royalty-free, irrevocable (except as stated in this section) patent license to make, have made, use, offer to sell, sell, import, and otherwise transfer the Work, where such license applies only to those patent claims licensable by You that are necessarily infringed by Your Contribution(s) alone or by combination of Your Contribution(s) with the Work to which such Contribution(s) was submitted. If any entity institutes patent litigation against You or any other entity (including a cross-claim or counterclaim in a lawsuit) alleging that your Contribution, or the Work to which you have contributed, constitutes direct or contributory patent infringement, then any patent licenses granted to that entity under this Agreement for that Contribution or Work shall terminate as of the date such litigation is filed.

4. You represent that you are legally entitled to grant the above licenses, including but not limited to any video content shared or recorded as related to collaboration with Zed and the Zed community. If your employer(s) has rights to intellectual property that you create that includes your Contributions, you represent that you have received permission to make Contributions on behalf of that employer, that your employer has waived such rights for your Contributions to Company, or that your employer has executed a separate Corporate CLA with Company.

5. You represent that each of Your Contributions is Your original creation (see section 7 for submissions on behalf of others). You represent that Your Contribution submissions include complete details of any third-party license or other restriction (including, but not limited to, related patents and trademarks) of which you are personally aware and which are associated with any part of Your Contributions.

6. You are not expected to provide support for Your Contributions, except to the extent You desire to provide support. You may provide support for free, for a fee, or not at all. Unless required by applicable law or agreed to in writing, You provide Your Contributions on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied, including, without limitation, any warranties or conditions of TITLE, NON-INFRINGEMENT, MERCHANTABILITY, or FITNESS FOR A PARTICULAR PURPOSE.

7. Should You wish to submit work that is not Your original creation, You may submit it to Company separately from any Contribution, identifying the complete details of its source (e.g., attribution) and of any license or other restriction (including, but not limited to, related patents, trademarks, and license agreements) of which you are personally aware, and conspicuously marking the work prior to submitting it to Zed or the Zed community.

8. You agree to notify Company of any facts or circumstances of which you become aware that would make these representations inaccurate in any respect.

**Comment by maxdeviant** [comment]
> It is currently not possible to read the Zed cla at https://zed.dev/cla without connecting with GitHub to Zed, and in that process handing over personal information such as Email without information about how it will be used.

Thanks for raising this, @Erk-.

We've made it possible to view the CLA without signing in (you will still need to sign in with GitHub if you wish to accept the CLA).
